tests/functional/lti_certification/v13/conftest.py

```diff
@@ -59,6 +59,7 @@ def application_instance(lti_registration):
         tool_consumer_instance_guid="TEST_CONSUMER_INSTANCE_GUID",
         lti_registration=lti_registration,
         deployment_id="testdeploy",
+        organization=factories.Organization(),
```

Comment:
These two seem to be required but I can't point exactly what change in this PR makes it necessary.

I think it working before was an accident of how / when we flush to the DB.